wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt

```diff
@@ -123,195 +122,218 @@ fun MainView(
+                                            onNavigationClicked(
+                                                mainViewModel.entitiesByDomain.mapKeys {
+                                                    mainViewModel.stringForDomain(
+                                                        it.key
+                                                    )!!
+                                                },
+                                                mainViewModel.entitiesByDomain.keys.map {
+                                                    mainViewModel.stringForDomain(
+                                                        it
+                                                    )!!
+                                                }.sorted()
+                                            ) { true }
                                         },
-                                        colors = ChipDefaults.primaryChipColors()
+                                        colors = ChipDefaults.secondaryChipColors()
                                     )
                                 }
                             }
                         }
-
-                        item {
-                            Spacer(modifier = Modifier.height(32.dp))
-                        }
-                        // All entities regardless of area
-                        if (mainViewModel.entities.isNotEmpty()) {
-                            item {
-                                Chip(
-                                    modifier = Modifier
-                                        .fillMaxWidth(),
-                                    icon = {
-                                        Image(
-                                            asset = CommunityMaterial.Icon.cmd_animation,
-                                            colorFilter = ColorFilter.tint(Color.White)
-                                        )
-                                    },
-                                    label = {
-                                        Text(text = stringResource(commonR.string.all_entities))
-                                    },
-                                    onClick = {
-                                        onTestClicked(
-                                            mainViewModel.entitiesByDomain.mapKeys { mainViewModel.stringForDomain(it.key)!! },
-                                            mainViewModel.entitiesByDomain.keys.map { mainViewModel.stringForDomain(it)!! }.sorted()
-                                        ) { true }
-                                    },
-                                    colors = ChipDefaults.secondaryChipColors()
-                                )
-                            }
-                        }
                     }
                 }
 
+                if (mainViewModel.isFavoritesOnly)
+                    item {
+                        Spacer(Modifier.padding(4.dp))
```

Comment:
The main list is currently using a [32dp spacer before the all entities/settings chips](https://github.com/home-assistant/android/blob/bd822afabad97aae8dfeb1d4cec5229216f63bb0/wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt#L284-L286), let's use the same size everywhere.